src/features/governance/updateProposalsInDB.ts

```diff
@@ -254,12 +279,12 @@ async function mergeProposalDataIntoPGRow({
     proposer: proposalQueuedEvent.args.proposer,
     deposit: BigInt(proposalQueuedEvent.args.deposit),
     networkWeight,
-    executedAt: metadata?.timestampExecuted ? metadata.timestampExecuted / 1000 : null,
     transactionCount: Number(numTransactions),
+    [column]: new Date(),
```

Comment:
sorry but if we are setting executedAt approvedAt it should be based on the timestamp of the block not when the webhook was received